### DISCUSSION TITLE
Can I  :  Change the pink Octopus for something that is more in line with what I need.... ( the 3D skull in Freetrack was coool...)

### DISCUSSION THREAD

**Discussion by 5YNTH3T1K** [opentrack/opentrack#1928] [Ideas]
Can I change the Octopus for something else?  If it's just a graphic file where is it hiding? 
Is it possible to make a 3D graphic like Free Track ? 

Cheers!

**Comment by GO63-samara** [comment]
https://github.com/opentrack/opentrack/discussions/1418

**Reply by 5YNTH3T1K** [reply]
I was about to search for "Lets talk about the pose-widget for a moment."  on a popular search engine... 

Saved !

**Comment by 5YNTH3T1K** [comment]
Oh, my, god. 

I am not reading that entire thread.  

A 3D head would be great.  A wire frame background is FINE. ( no it does not make me think of prison cells, why would it ???) 
Just turn OFF the 3D animation when the window focus is lost ?  It turns back on when you re centre and focused. 

wow my PC is ancient.... 
	Name			Intel Core i5 4670K
	Codename		Haswell
	Specification		Intel(R) Core(TM) i5-4670K CPU @ 3.40GHz

and yet... it still runs !  ( windows 7 ...)

So, I guess nothing came of that thread ??? Whats the bottom line ?

**Reply by 5YNTH3T1K** [reply]
and time passes...

So I am picking the answer to this question is a simple NO and/but a complicated YES.  

Grinding my way through a conversation between two people just to find the answer to 
my question that could have been answered by now if one of those people had taken 
a tiny bit longer to type out a reply. 

and reading a disscusion where two parties argue about what kind of other graphic 
should be used... is eyerolling material. 

If there is an easy way to change the graphic then please let me know. I can then make my own. 

Cheers.
